code.rb

```diff
@@ -150,13 +150,45 @@
   event << ' `^thanks`: Thanks to these radical donors!'
   event << ' `^feedback <words>`: Want to tell me my bot sucks without going into the CB Server? Well, here you go.'
   event << ' `^noot`: noot'
+  event << ' `^serverinfo` : Want to know about where chu are, well you know what this is for then.'
+end
+
+bot.command(:serverinfo, max_args: 1) do |event|
```

Comment:
Should be 0, unless we're going to be doing some weird thing with args :man_shrugging: